src/test/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " Excavator started its work.");
```

Comment:
The message in the `doWork` method should be more informative and indicate what type of `Machine` is working. Consider rephrasing the message to clearly state that an Excavator is starting its work. For example: 'Excavator is starting its work.'